### PR TITLE
Add franklin templeton to captive core QS

### PR DESCRIPTION
**Pull request by mollykarcher**
<details>
  <summary>PR Checklist</summary>
  
### PR Structure

* [ ] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
* [ ] This PR avoids mixing refactoring changes with feature changes (split into two PRs
  otherwise).
* [ ] This PR's title starts with name of package that is most changed in the PR, ex.
  `services/friendbot`, or `all` or `doc` if the changes are broad or impact many
  packages.

### Thoroughness

* [ ] This PR adds tests for the most critical parts of the new functionality or fixes.
* [ ] I've updated any docs ([developer docs](https://developers.stellar.org/api/), `.md`
  files, etc... affected by this change). Take a look in the `docs` folder for a given service,
  like [this one](https://github.com/stellar/go/tree/master/services/horizon/internal/docs).

### Release planning

* [ ] I've updated the relevant CHANGELOG ([here](services/horizon/CHANGELOG.md) for Horizon) if
  needed with deprecations, added features, breaking changes, and DB schema changes.
* [ ] I've decided if this PR requires a new major/minor version according to
  [semver](https://semver.org/), or if it's mainly a patch change. The PR is targeted at the next
  release branch if it's not a patch change.
</details>

### What

Add Franklin Templeton to quorum set for captive core nodes. Actual config is managed in puppet and updated [here](https://github.com/stellar/puppet-v4/pull/3157), but we should keep this example in-line as partners often look to it as an example when setting up their config. Additional context in [slack](https://stellarfoundation.slack.com/archives/C02B04RMK/p1677884495564779).

### CODE REVIEW

File: services/horizon/configs/captive-core-pubnet.cfg [hunk 1]
```diff
@@ -165,4 +169,25 @@ NAME="Blockdaemon_Validator_3"
...
+PUBLIC_KEY = "GCMSM2VFZGRPTZKPH5OABHGH4F3AVS6XTNJXDGCZ3MKCOSUBH3FL6DOB"
+ADDRESS = "stellar2.franklintempleton.com:11625"
+HISTORY = "curl -sf https://stellar-history-usc.franklintempleton.com/azuscshf401/{0} -o {1}"
+
+[[VALIDATORS]]
+NAME = "FT_SCV_3"
+HOME_DOMAIN = "www.franklintempleton.com"
+PUBLIC_KEY = "GA7DV63PBUUWNUFAF4GAZVXU2OZMYRATDLKTC7VTCG7AU4XUPN5VRX4A"
+ADDRESS = "stellar3.franklintempleton.com:11625"
+HISTORY = "curl -sf https://stellar-history-ins.franklintempleton.com/azinsshf401/{0} -o {1}"
```
Comment (Shaptic) [inline services/horizon/configs/captive-core-pubnet.cfg:193]: sanity check - these three history URLs are **supposed** to be slightly different, right?

**Review by Shaptic — COMMENTED** [review]
We should add this to the docker configs, as well: https://github.com/stellar/go/tree/master/services/horizon/docker.

File: services/horizon/configs/captive-core-pubnet.cfg (hunk 1 above)
Comment (mollykarcher) [reply services/horizon/configs/captive-core-pubnet.cfg:193]: Yeah, they are

**Review by Shaptic — APPROVED** [review]
yeet it

**Pull request merged**